crates/typst/src/eval/version.rs

```diff
@@ -0,0 +1,223 @@
+use crate::diag::{bail, StrResult};
+use crate::eval::Value;
+use ecow::EcoVec;
+use serde::{Serialize, Serializer};
+use std::cmp::Ordering;
+use std::fmt::{Debug, Display, Formatter};
+use std::hash::{Hash, Hasher};
+use std::iter::repeat;
+use std::str::FromStr;
+use typst::eval::Array;
```

Comment:
should be split into std - blank line - extern - blank line - crate. within the `typst` crate prefer `super` and `crate` over `typst`, i.e. `use super::Array`.